base/titer_model.py

```diff
@@ -142,10 +141,19 @@ def filter_strains(titers, strains):
         0
         """
         return {key: value for key, value in titers.iteritems()
-                if key[0] in strains or key[1][0] in strains}
+                if key[0] in strains and key[1][0] in strains}
```

Comment:
yes, all titer model are based on a direct comparison between the sequences of the reference and test virus (either via a path in the tree or by detecting mutations). if one of them is absent, the current titer model implementations can't use the measurement.  Autologous/homologous titers are a different issue. this is about a point of comparison for individual measurements (a titer drop relative to the titer the serum has against the virus it was raised against). We can do without autologous  titers by approximating them with the maximal titers. but this is rarely an issue since almost all sera are measured multiple times against "their" virus.